src/main/java/taxi/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,120 @@
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT * FROM manufacturer "
+                + "WHERE manufacturer_id = ? and isDeleted = false ";
+        Manufacturer manufacturer = null;
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if (resultSet.next()) {
+                manufacturer = createManufacturerFromDb(resultSet);
+            }
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get manufacturer with "
+                    + id + " id from DB", ex);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String query = "SELECT * FROM manufacturer";
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                manufacturerList.add(createManufacturerFromDb(resultSet));
+            }
+            return manufacturerList;
+
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get all elements ", ex);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacturer "
+                + "SET manufacturer_name = ?,manufacturer_country = ? "
+                + "WHERE manufacturer_id = ? and isDeleted = false";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, manufacturer.getModel());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setLong(3, manufacturer.getId());
+            preparedStatement.executeUpdate();
+            preparedStatement.close();
+            return manufacturer;
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't update element", ex);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE manufacturer SET isDeleted = true WHERE manufacturer_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setLong(1, id);
+            int numberOfDeletedManufacturer = statement.executeUpdate();
+            statement.close();
+            return numberOfDeletedManufacturer > 0;
```

Comment:
```suggestion
            return statement.executeUpdate() > 0;
```
no need to close statement, because it's the one and only statement in this method